### DISCUSSION TITLE
Import saves and savestates

### DISCUSSION THREAD

**Discussion by trymeouteh** [Swordfish90/Lemuroid#438] [Q&A]
I am unable to copy and paste files into the /Android/data/... folder with my default file manager app on my device. Is there a way to import my saves and savestates into Lemuroid>

**Comment by thewalkerz** [comment]
I have the same issue, in my case it is because I have Android 11 and due to the new security policies it does not allow some file explorers to show this folder, the lemuroid team should move this folder to another accessible location. I also can't use synching to sync my files between devices, it won't let me access this folder anymore. 😢

**Reply by LTDDAD** [reply]
You can use this one https://github.com/SimpleMobileTools/Simple-File-Manager
Its FOSS and its work on my pixel 3a grapheneos phone. I CAN transfer data from "data" folder with this app. Install it and try. Will work. @trymeouteh @amzony

**Reply by catguy323** [reply]
> You can use this one https://github.com/SimpleMobileTools/Simple-File-Manager Its FOSS and its work on my pixel 3a grapheneos phone. I CAN transfer data from "data" folder with this app. Install it and try. Will work. @trymeouteh @amzony

"there was a problem parsing the  package"

**Comment by catguy323** [comment]
i moved it but it did not use my save file

**Comment by DilanOrmisiarAvask** [comment]
I use adb (Android Debug Bridge) to do this, you only need some simple commandline knowledge:
https://developer.android.com/tools/adb

**Reply by catguy323** [reply]
> I use adb (Android Debug Bridge) to do this, you only need some simple commandline knowledge: https://developer.android.com/tools/adb

i got adb installed

**Comment by benjaminoakes** [comment]
You may be able to use a file browser on your computer to do this.  Use a USB cable to connect the phone to the computer, allow MTP, and then browse to the correct directory.
